pyp2rpm/utils.py

```diff
@@ -211,3 +134,25 @@ def c_time_locale():
     locale.setlocale(locale.LC_TIME, 'C')
     yield
     locale.setlocale(locale.LC_TIME, old_time_locale)
+
+
+def rpm_eval(macro):
+    """Get value of given macro using rpm tool"""
+    try:
+        value = subprocess.Popen(
+            ['rpm', '--eval', macro],
+            stdout=subprocess.PIPE).communicate()[0].strip()
+    except OSError:
+        logger.error('Failed to get value of {0} rpm macro'.format(
```

Comment:
I think this should be *warning* for the user, also saying that the default path will be used.